test/functional/update_v2/update-iterative-and-full.bats

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bats
+
+load "../testlib"
+
+test_setup() {
+
+	create_test_environment "$TEST_NAME"
+	create_bundle -L -n test-bundle-1 -f /foo/test-file1 "$TEST_NAME"
+	create_bundle -L -n test-bundle-2 -f /foo/test-file2 "$TEST_NAME"
+	create_version "$TEST_NAME" 20 10
+	update_bundle -i "$TEST_NAME" test-bundle-1 --update /foo/test-file1
+	create_version "$TEST_NAME" 30 20
+	update_bundle -i "$TEST_NAME" test-bundle-1 --update /foo/test-file1
+	update_bundle -i "$TEST_NAME" test-bundle-2 --update /foo/test-file2
```

Comment:
Which bundle doesn't use iterative manifests? Do you need to drop the '-i' on one of them?

Reply:
We need to use the '-i' option to tell the testlib to create an iterative manifest for the updates.
But since we are updating from version 10 to version 30, the following iterative manifests are going to exist in web-dir/30:
- Manifest.test-bundle-1.I.20 (since we are updating from 10 -> 30 this iterative manifest doesn't apply so the full manifest will be used instead)
- Manifest.test-bundle-2.I.10

Reply:
Thanks, I understand now.